archive/dto.ksy

```diff
@@ -0,0 +1,79 @@
+meta:
+  id: android_dto
```

Comment:
Also please rename the file accordingly: ~`archive/dto.ksy`~ &xrarr; **`archive/android_dto.ksy`**.